lib/src/foundation/feature_discovery.dart

```diff
@@ -37,6 +37,16 @@ class FeatureDiscovery extends StatelessWidget {
   /// call [completeCurrentStep] instead.
   static void dismiss(BuildContext context) => Bloc.of(context).dismiss();
 
+  /// This return the feature id of the current feature discovery step, i.e.
+  /// of the [DescribedFeatureOverlay] that is currently supposed to be shown, or `null`.
+  ///
+  /// Note that this will also return the feature id of the current step of the steps
+  /// you passed to [discoverFeature] even when there is no [DescribedFeatureOverlay]
+  /// in the tree to display the overlay.
+  /// This means that you cannot use this to check if a feature overlay is being displayed.
+  static String activeFeatureId(BuildContext context) =>
```

Comment:
In case you are wondering: this is not used internally, but users of the package might want to access this from the outside.

I certainly need it in my usage of the package (I explained it in the PR comment).